tests/should-fail/iati-activities/iati-activity/tag/err-not-iati-xml-missing-attribute.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity> <!-- at least 1 -->
+    <iati-identifier></iati-identifier>  <!-- 1 and only 1-->
```

Comment:
Lots of comments like this that should be removed.

Will do a more thorough look once comments are removed (it's hard to do when they're there!)

Reply:
6be59361a38f36b706476abc63a4b30f516afd31